common/metadata-helper.h

```diff
@@ -25,7 +25,7 @@ namespace rs2
 
         static bool can_support_metadata(const std::string& product)
         {
-            return product == "D400";
+            return product == "D400" || product == "D500";
```

Comment:
Do we have supported products that don't support metadata? Platform camera?

Reply:
I guess that's the only one that's left...